File: it/test-code/expected/suite/csharp/Test/Entry.cs
using Newtonsoft.Json;
using System;
using System.Text;

namespace Test {
  [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
  public class Entry {
    [JsonConstructor]
    public Entry() {
    }

    public override Int32 GetHashCode() {
      Int32 result = 1;
      return result;
    }

    public override Boolean Equals(Object other) {
      Entry o = other as Entry;

      if (o == null) {
        return false;
      }

      return true;
    }

    public override String ToString() {
      StringBuilder b = new StringBuilder();

      b.Append("Entry");
      b.Append("(");
      b.Append(")");

      return b.ToString();
    }
  }
}

File: it/structures/java/code-default/test/Entry.java
package test;

import com.fasterxml.jackson.annotation.JsonCreator;

public class Entry {

    @JsonCreator
    public Entry() {}

    @Override
    public String toString() {
        return "Entry()";
    }

    @Override
    public int hashCode() {
        int result = 1;
        final StringBuilder b = new StringBuilder();
        return result;
    }

    @Override
    public boolean equals(final Object other_) {
        if (other_ == null) {
            return false;
        }

        if (!(other_ instanceof Entry)) {
            return false;
        }

        @SuppressWarnings("unchecked")
        final Entry o_ = (Entry)other_;

        return true;
    }

    public static class Builder {

        private Builder() {}

        public Entry build() {

            return new Entry();
        }
    }

    /**
     * Construct a new builder.
     */
    public static Builder builder() {
        return new Builder();
    }
}
